docs/changelog/2024/2406.md

```diff
@@ -0,0 +1,121 @@
+---
+  title: Cycle 2406 (13 Jun)
+---
+
+--8<-- "includes/abbreviations.md"
+
+This cycle is a major release, with a significant amount of changes. A lot of these changes come from the renaming of waypoints, under direction from ICAO.
+
+## SOP Changes
+
+### NZCH: Christchurch
+
+Amendment to the in-progress CH TMA SOPs for the introduction of the `BELEE #J` and `BELEE #K` STARs, and when to use them over the `BELEE #A` and `BELEE #B` STARs.
+
+----
+
+## Data Changes
+
+### Aerodrome
+
+#### NZAA: Auckland
+
+- The following STARs have been amended and re-sequenced: `EXOPI 2B`, `BASIV 9B`, `EXOPI 8D`, `RIKNI 2N`, `EXOPI 9A`, `RAXIN 3Q`, `MEMOR 3P`.
+    - Waypoint `SELKA` replaced by `ELPAK`.
```

Comment:
Should we put the waypoint changes first and make it a bit more explicit and then flow on with the STARs just to make it nice and clear those two waypoints are no longer? (Similar to what has been done with NZCH)

Reply:
Sounds like a good idea to me! Done in the next commit.